roxie/udplib/udpaeron.cpp

```diff
@@ -81,13 +100,10 @@ int startAeronDriver()
         context->warn_if_dirs_exist = false;
         context->term_buffer_sparse_file = false;
 
-        // MORE - should possibly allow these to be configured, or experiment to find what values work well for Roxie
-        // In my (very rudimentary and non-representative) tests these values seem ok.
-
-        context->mtu_length=16384;
-        context->socket_rcvbuf=2097152;
-        context->socket_sndbuf=2097152;
-        context->initial_window_length=2097152;
+        if (aeronMtuLength) context->mtu_length = aeronMtuLength;
+        if (aeronSocketRcvbuf) context->socket_rcvbuf = aeronSocketRcvbuf;
+        if (aeronSocketSndbuf) context->socket_sndbuf = aeronSocketSndbuf;
+        if (aeronInitialWindow) context->initial_window_length = aeronInitialWindow;
```

Comment:
Its ok to not set these by default ?  Are the default values reasonable ?

Reply:
Default values should be reasonable (they are hand-picked by the aeron developers, who probably have a better guess at what is reasonable than I do!)